### PR TITLE
deploy: Upload source bundles

### PR DESCRIPTION
**Pull request by tonyo**
Should work after a new version of sentry-cli  (>1.47.1) is released.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -55,28 +55,32 @@ RUN echo "Building OpenSSL" \
...
 ### Builder stage ###
 #####################
 
+FROM getsentry/sentry-cli:1.47.1 AS sentry-cli
 FROM semaphore-deps AS semaphore-builder
 
+COPY --from=sentry-cli /bin/sentry-cli /bin/sentry-cli
 COPY . .
 
+# TODO remove this
```
Comment (jan-auer) [inline Dockerfile]: Is this TODO still relevant?

File: Dockerfile [hunk 2]
```diff
@@ -55,28 +55,32 @@ RUN echo "Building OpenSSL" \
...
     && make depend \
-    && make && make install \
+    && make -j$(nproc) && make install \
     && cd .. && rm -rf openssl-$OPENSSL_VERS.tar.gz openssl-$OPENSSL_VERS checksums.txt
 
 #####################
 ### Builder stage ###
 #####################
 
+FROM getsentry/sentry-cli:1.47.1 AS sentry-cli
```
Comment (jan-auer) [inline Dockerfile]: Can we move this to a later stage and then use `:1` here, just to make sure we get all bugfix releases in here? That way we wouldn't prune build caches in cases sentry-cli changes, but still get updates.

Comment (tonyo) [reply Dockerfile]: Can change to `:1`, but this should go before the next line, that sets the current build context to `semaphore-deps` (and we need it for the build itself).

File: Dockerfile (hunk 1 above)
Comment (tonyo) [reply Dockerfile]: Nope :)

File: Dockerfile (hunk 2 above)
Comment (tonyo) [reply Dockerfile]: Regarding cache pruning: AFAIK we don't always pull for the latest image version, but let me confirm.

**Review by jan-auer — APPROVED** [review]

**Pull request merged**